.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,23 @@
-<!--
-Please follow the naming conventions in the "guidelines for contributing" link above or the pull request may be rejected.
+Fixes #
 
-Also, do consider outlining the solution taken as doing so will likely help in the reviewing process
-(e.g., when the pull request involves non-trivial changes)
--->
+**PR Checklist**
 
-Fixes #
+<!-- Remove this portion after you have made the checks. -->
+
+Ensure that you have:
+- [ ] Read and understood our PR guideline: https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr
+  - [ ] Added the issue number to the "Fixes" keyword above
+  - [ ] Titled the PR as specified in the abovementioned document
+- [ ] Made your changes on a branch other than `master` and `release`
+- Gone through all the changes in this PR and ensured that:
+  - [ ] No unintended changes were made
+  - [ ] They addressed one (and only one) issue
+- [ ] Run and passed static analysis: `./gradlew staticAnalysis`
+- [ ] Added/updated tests, if changes in functionality are involved
```

Comment:
Should it be `were involved` instead of `are involved` since the other points all refer to the changes using the past tense?